### ISSUE TITLE
Question about the accuracy

### ISSUE THREAD

**Issue by ujeong1** [MichailChatzianastasis/GOAT#1]
Hi,

I've run the program using the following command, as suggested in README,
and I find the accuracy is not close to 92.11 which is reported as the accuracy of Disease_nc Dataset.

Could you guide me if I misunderstood anything?

- **Command** : python train.py --goat True
- **Arguments**: Namespace(adsf=False, aggregator_type='lstm', alpha=0.2, central=False, cuda=True, dataset='disease_nc', dropout=0.5, edge_index=False, edge_prob=0.01, epochs=1000, fastmode=False, final_mlp=False, final_outd=8, gcn=False, gin=False, goat='True', hidden=32, hidden_2=8, impl4=False, lr=0.005, lstm_h1=8, lstm_h2=8, mlp=False, nb_heads=1, nb_heads_2=1, no_cuda=False, node_batch=False, number_of_classes=2, number_of_nodes=1000, outd_1=8, outd_2=8, patience=250, pna=False, pooling_1='cat', pooling_2='cat', random=False, random_idx=0, rnn_agg='lstm', sage=False, seed=72, shared=False, sparse=False, v2=False, v3=False, weight_decay=0.0005)
- **Output**: Test set results: loss= 0.4856 accuracy= 0.7943

**Comment by MichailChatzianastasis** [comment]
Hello,

By running python train.py --goat True (without tuning), I obtain 91.55 accuracy (from epoch 855th) which is close to the reported accuracy of the paper.
I would suggest to clone the repository again, in case you have an older version.
